health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -398,56 +420,61 @@ const createUpdatePlanProject = async (req) => {
           return updatedRow;
-        })
-        
+        });
+        const formulasToUpdate = totalFormData?.FORMULA_CONFIGURATION?.formulaConfiguration?.formulaConfigValues
+          ?.filter((row) => {
+            return row.category && row.output && row.input && row.operatorName && row.assumptionValue;
+          })
+          ?.map((row) => {
+            const updatedRow = { ...row };
+            const operatorName = row?.operatorName;
+            delete updatedRow?.operatorName;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)